backend/src/notes/notes.service.ts

```diff
@@ -148,15 +162,17 @@ export class NotesService {
   private createGroupPermission(
     note: Note,
     group: Group,
-    permission: DefaultAccessPermission,
+    accessLevel: DefaultAccessLevel,
   ): NoteGroupPermission | null {
-    return permission === DefaultAccessPermission.NONE
-      ? null
-      : NoteGroupPermission.create(
-          group,
-          note,
-          permission === DefaultAccessPermission.WRITE,
-        );
+    if (accessLevel === DefaultAccessLevel.NONE) {
+      return null;
+    } else {
+      return NoteGroupPermission.create(
+        group,
+        note,
+        accessLevel === DefaultAccessLevel.WRITE,
+      );
+    }
```

Comment:
The else branch does not require parentheses as the if-condition already does a return.
Removing the parentheses makes the code more readable.

Reply:
In this case I would prefer a ternary operator or keep the indentation so both returns are on one level.